internal/svcctl/svcctl.go

```diff
@@ -50,19 +58,19 @@ func EnsureExecStartedAndLocateHTTP(ipComm IPCommunicator, exec string) (addr st
 	addr, err = LocateHTTP(ipComm)
 	if err != nil {
 		if !errs.Matches(err, &ipc.ServerDownError{}) {
-			return "", errs.Wrap(err, "Cannot locate HTTP port of ipc server")
+			return "", err
 		}
 
 		ctx, cancel := context.WithTimeout(context.Background(), commonTimeout)
 		defer cancel()
 
 		if err := startAndWait(ctx, ipComm, exec); err != nil {
-			return "", errs.Wrap(err, "Cannot start ipc server at %q", exec)
+			return "", err
 		}
 
 		addr, err = LocateHTTP(ipComm)
 		if err != nil {
-			return "", errs.Wrap(err, "Cannot locate HTTP port of ipc server after start succeeded")
+			return "", err
```

Comment:
This doesn't seem like an improvement?

Reply:
Same as the message in StopServer, these messages are mostly redundant, but the "after start" is very useful, so I've included that again.

Reply:
It's not just about the errors being useful, it's also about tracing where the error came from. You advocated we removed stack traces from errors, which we did because we should effectively get these from the error wrapping anyway. This is undermining that decision.